app/src/main/res/values-de/strings.xml

```diff
@@ -167,6 +167,7 @@
     <!-- News -->
     <string name="no_link_existing">Kein Link gefunden.</string>
     <string name="info_title">Neuigkeiten:</string>
+    <string name="open_options_popup_description">Optionen Popup Öffnen</string>
```

Comment:
What do you think about this alternative translation (the grammar feels kind of broken otherwise)?

```suggestion
    <string name="open_options_popup_description">Popup-Fenster mit Optionen öffnen</string>
```